common/proc_allowing-external-access-knative-services-OCP-4x.adoc

```diff
@@ -0,0 +1,48 @@
+// This module is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+
+[id='allowing-external-access-knative-services-OCP-4x_{context}]
+= Allowing external access to Knative services on OCP 4.0
+
+Use a configured hostname to access Knative services on the OpenShift 4.0 cluster
+
+.Procedure
+
+. Updating Knative's config-domain ConfigMap and replacing "example.com" with your domain's suffix.
```

Comment:
`config-domain` and `example.com`? Or how do we format these types of options / names for other docs?

Also, present tense? i.e. "Update the Knative `config-domain` ConfigMap and replace `example.com` with your domain's suffix."

Reply:
I think for mod docs guidelines, procedures have to start with a gerund. I know this format i a bit funny, but I think William identifies them as two procedures. What do you think? Is the gerund "rule" not really applicable here?

Reply:
Can you give an example of the gerund?

Reply:
'Updating' is the subject, so is acting like a gerund in this case.
[Some examples](https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/content/topics/module_mod-docs-procedure-examples.adocurl). I'm not too sure if it applies here as well. William Tam identified them as two separate procedures, so that's why I thought it should start with a gerund. We can make it two separate procedure modules if that works better.

Reply:
Yeah that makes sense, thanks for looking at this :+1: I'd probably keep two procedure modules just to keep things simpler as the project gets more complicated :)